### DISCUSSION TITLE
Which version of velero is tested and compatible with K8s 1.30 ?

### DISCUSSION THREAD

**Discussion by navilg** [vmware-tanzu/velero#7917] [Community support Q&A]
Hello Team,

I am planning to upgrade K8s cluster to 1.30. I do not see any tested/compatible version of velero on K8s 1.30 in compatibility matrix.
Do you have any planned release of velero which is tested and compatible with K8s 1.30 ?

@Lyndon-Li @reasonerjt 

Thanks

**Comment by blackpiglet** [comment]
We didn't have the Velero tested with k8s v1.30 yet.
The most recently released Velero v1.14.0 was tested against v1.29.

Although we haven't tried it yet, I suggest you use v1.14.0 with k8s v1.30.

**Reply by navilg** [reply]
Thanks @blackpiglet . is there any plan to test 1.14.0 on K8s 1.30 ?

**Reply by navilg** [reply]
It looks like helm chart for 1.14.0 is not yet released.

**Reply by blackpiglet** [reply]
There is no such plan yet, but the v1.15.0 will definitely be tested against v1.30 k8s.
Ah, right. I will check with the helm chart owner.

**Reply by blackpiglet** [reply]
There is already a PR in progress.
https://github.com/vmware-tanzu/helm-charts/pull/590
